tools/nemo_forced_aligner/requirements.txt

```diff
@@ -0,0 +1 @@
+nemo
```

Comment:
Its nemo_toolkit['all']

Reply:
I have changed it to nemo_toolkit[asr] and tried reinstalling and running align.py - worked for me so will mark this as resolved.